### PR TITLE
event and user models adedd

### PR DESCRIPTION
**Pull request by Tek58**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -7,5 +7,24 @@ generator client {
   provider = "prisma-client-js"
 }
 
+model Event {
+  id            Int       @default(autoincrement()) @id
+  title         String
+  description   String?
+  location      String
```
Comment (simbamentor) [inline prisma/schema.prisma]: can be optional

File: prisma/schema.prisma [hunk 2]
```diff
@@ -7,5 +7,24 @@ generator client {
...
+  startTime     DateTime
+  endTime       DateTime
+  createdBy     User?       @relation(fields: [userId], references: [id])
```
Comment (simbamentor) [inline prisma/schema.prisma]: can you call it `owner`? also external guests can create events. the owner will always be the owner of the event, the external guest that creates the event can also be a participant

File: prisma/schema.prisma (hunk 1 above)
Comment (Tek58) [reply prisma/schema.prisma]: Done

File: prisma/schema.prisma (hunk 2 above)
Comment (Tek58) [reply prisma/schema.prisma]: Changed

**Pull request merged**